pkg/restapi/issuer/operation/operations.go

```diff
@@ -1728,11 +1740,30 @@ func (o *Operation) hanlDIDExStateMsg(msg service.StateMsg) error {
 	return nil
 }
 
-// read credential manifest from profile URL endpoints
-// TODO for now returning empty manifest, TO BE IMPLEMENTED [issue##561 & issue#563]
-func (o *Operation) readCredentialManifest() *cm.CredentialManifest {
+// Read credential manifest issuer detail from persisted profile data and scope from persisted transaction cred scope.
+// cm.Issuer's ID will be used as issuer ID which identifies who the issuer of the credential(s) will be.
+// Credential Manifest Styles represents an Entity Styles object as defined in credential manifest spec.
+// TODO issue#561 Add credential manifest presentation definition
+func (o *Operation) readCredentialManifest(profileData *issuer.ProfileData,
+	txnCredScope string) *cm.CredentialManifest {
+	if cmDescriptor, ok := o.cmOutputDescriptor[txnCredScope]; ok {
+		return &cm.CredentialManifest{
+			ID:      uuid.NewString(),
+			Version: credentialManifestVersion,
+			Issuer: cm.Issuer{
+				ID:     profileData.IssuerID,
```

Comment:
I don't see anywhere these being set profile data??
Where is profile creation step which accepts these 2 additional params - `IssuerID & CMStyle`?

Reply:
https://github.com/trustbloc/adapter/blob/3cbdf9ec92b3cb7a35ff8bf118b40ba19adcbad6/pkg/restapi/issuer/operation/operations.go#L2102 Its being set here

Reply:
@talwinder50 I meant how an issuer can pass it? I don't see these as part of profile creation request

Reply:
@sudeshrshetty It is mentioned here https://github.com/trustbloc/adapter/blob/33e8e045be4cdf7217bc148e6a0342e33d911c36/pkg/restapi/issuer/operation/models.go#L31